index.js

```diff
@@ -21,10 +21,6 @@ module.exports = user => {
 		username: res.body.user.username,
 		website: res.body.user.external_url || ''
 	})).catch(err => {
-		if (err.statusCode === 404) {
-			return 'Not found!';
-		} else {
-			return 'Error!'
-		}
+		return err;
```

Comment:
Alright, makes sense. Yeah it does seem better to return undefined. Anything is better than getting an unhandled promise rejection.